AmahiAnywhere/AmahiAnywhere/Presentation/Files/FilesViewController.swift

```diff
@@ -538,9 +538,26 @@ extension FilesViewController: UINavigationControllerDelegate, UIImagePickerCont
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
         picker.dismiss(animated: true, completion: nil)
         
-        guard let _ = info[.editedImage] as? UIImage else {
+        guard let image = info[.editedImage] as? UIImage else {
             return
         }
+
+        var url: String = ""
+        if directory != nil {
+            url = ServerApi.shared!.getFileUri(directory!)!.absoluteString
+        }
+        else {
+            url = ServerApi.shared!.getShareUri(share)!.absoluteString
+        }
+        Network.shared.uploadFile(url, data: image.jpegData(compressionQuality: 100)!, fileName: "IMG.jpeg") { success in
```

Comment:
I used static name `IMG.jpeg` so that we would get file names as IMG.jpeg, IMG_xyz.jpeg (where xyz is a number, this when IMG already exists), IMG_abc.jpeg etc.
I couldn't get the name of the selected file, from the file I get something in the format "some_number.jpeg", which is not the file name. 
Or should I pass that "some_number.jpeg" in place of `IMG.jpeg`?

Reply:
What exactly is `xyz` or `abc` ??

Reply:
He means on the first upload the filename will be like `IMG.jpeg` then after that, it will be `IMG_yyyymmdd-HHMM.jpeg` format. This file renaming is handled by our fs when it gets a file with the same name already present in the directory but different md5 hash. Here `yyyymmdd-HHMM` is what he was referring to as `xyz` or `abc`.

Reply:
@iswiftshek ideally the file name should be kept the same as it was on user's iPhone. Is there no way to get the original filename on the device?

Reply:
Right, that should be done. But the last time I looked into this, I couldn't find a way around it. But to the best of what I remember there was one method but it was deprecated. 
I'll look into it again.

Reply:
Couldn't find a way to get the file name for images and videos picked from UIPickerViewController.
However, in case of documents, we get the file name directly from the url.